docs/modules/ROOT/pages/includes/quarkus-langchain4j-infinispan.adoc

```diff
@@ -10,23 +10,6 @@ h|[[quarkus-langchain4j-infinispan_configuration]]link:#quarkus-langchain4j-infi
 h|Type
 h|Default
 
-a|icon:lock[title=Fixed at build time] [[quarkus-langchain4j-infinispan_quarkus-langchain4j-infinispan-client-name]]`link:#quarkus-langchain4j-infinispan_quarkus-langchain4j-infinispan-client-name[quarkus.langchain4j.infinispan.client-name]`
```

Comment:
How come this is disappearing from the docs when we didn't remove the related property?

Reply:
And the same question for huggingface and BAM docs

Reply:
I have no idea whatsover

Reply:
I rebased and it looks better now